### PR TITLE
feat(config): add ThermoFloor z-temp2 thermostat

### PR DESCRIPTION
**Pull request by ivdimitro**
Added support for HeatIt Z-Temp2 battery operated thermostat

### CODE REVIEW

File: packages/config/config/devices/0x019b/z-temp2.json [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+			"maxValue": 50,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": {
+				"label": "Humidity reporting on change is disabled",
+				"value": 0
+			}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-temp2.json]:
```suggestion
			"options": [
				"label": "Humidity reporting on change is disabled",
				"value": 0
			]
```

File: packages/config/config/devices/0x019b/z-temp2.json [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+			"maxValue": 50,
+			"defaultValue": 10,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": {
+				"label": "Temperature reporting on change is disabled",
+				"value": 0
+			}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/z-temp2.json]:
```suggestion
			"options": [
				"label": "Temperature reporting on change is disabled",
				"value": 0
			]
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
Did you write the file manually or import from OZW? Specifically the malformed options arguments?

**Comment by AlCalzone** [comment]
Please disregard this comment. I'm trying to debug our bot
@zwave-js-bot fix lint

**Comment by ivdimitro** [comment]
I first imported the config from OZW and then change the wording a bit and added the malformated "options". I fixed the "options" array but it failed in the index generation. I think this is not due to this configuration file in particular.

**Comment by AlCalzone** [comment]
>  I think this is not due to this configuration file in particular.

It is, you mixed up min and max value on parameter #5 (or it is already wrong in the OZW file)

grafik

**Review by AlCalzone — CHANGES_REQUESTED** [review]
see other comment

**Comment by ivdimitro** [comment]
Finally all green. Sorry for the wasted time.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**